### PR TITLE
Add more wpa-supplicant features

### PR DESCRIPTION
**Pull request by ConnorRigby**
```elixir
:ok = request(name, {:REMOVE_NETWORK, :all})
netid = request(pid, :ADD_NETWORK)
:ok = request(pid, {:SET_NETWORK, netid, :ssid, "test"})
:ok = request(pid, {:SET_NETWORK, netid, :"key_mgmt", :NONE})
:ok = request(pid, {:SET_NETWORK, netid, :mode, 2})
:ok = request(pid, {:SELECT_NETWORK, netid})
```

### CODE REVIEW

File: Config.in [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Add project-specific packages for Buildroot here
+#
+# If these are non-proprietary, please consider contributing them back to
+# Nerves or Buildroot.
```
Comment (fhunleth) [inline Config.in]: I don't think that you meant to include `Config.in` in this commit.

Comment (ConnorRigby) [reply Config.in]: Damnit. :hourglass:

Comment (fhunleth) [reply Config.in]: I'm good with PR if `Config.in` is removed. There's probably an argument for checking it in, but for a different PR.

Comment (ConnorRigby) [reply Config.in]: Yeah this one gets me every time.

**Pull request merged**